pkg/util/randident/namegen.go

```diff
@@ -58,7 +58,7 @@ var escGenerators = []func(s *strings.Builder, r *rand.Rand){
 	func(s *strings.Builder, r *rand.Rand) { fmt.Fprintf(s, `%%%02x`, r.Int31n(256)) },
 	// SQL escape sequences.
 	func(s *strings.Builder, r *rand.Rand) { fmt.Fprintf(s, `\\u%04X`, r.Int31n(65536)) },
-	func(s *strings.Builder, r *rand.Rand) { fmt.Fprintf(s, `\\U%08X`, r.Uint32()) },
+	func(s *strings.Builder, r *rand.Rand) { fmt.Fprintf(s, `\\U%08X`, r.Int31n(0x10FFFF)) },
```

Comment:
The unicode package has a MaxRune constant if you felt like using it, but given the surrounding context I think this is clear enough.